lib/recurly.js

```diff
@@ -35,6 +36,10 @@ const defaults = {
   publicKey: '',
   parent: true,
   cors: false,
+  fraud: {
+    dataCollector: false,
```

Comment:
I think that name is pretty meaningful. It's called data collector.  Here is the page from the Kount docs that all of our merchants would have went through when setting up their account with Kount.

It doesn't map to any configuration in the app because those are separate things. The app does it's piece independent of this and works the same whether this is in effect or not. This is simply an additional feature we're allowing them to enable that will give them more info on the Kount side.

![screen shot 2016-05-11 at 9 36 51 am](https://cloud.githubusercontent.com/assets/405400/15182757/112ff4d8-175c-11e6-8f00-a03d0f22ef90.png)